ksy_schema.json

```
@@ -216,7 +224,29 @@
             }
           }
         },
-        "license": { "type": "string" },
+        "tags": {
+          "type": "array",
+          "description": "list of some identifiers that can give additional information about format. `lowercase-kebab-case` is preferred for names, as well as keeping the list sorted. You can find more examples in existing schemas at https://formats.kaitai.io/. Should be used only at top-level",
```

I would

1. start a new paragraph (`\n\n`) when introducing a new idea,
2. use a simple `should be` instead of complex structures like `is preferred (...), as well as (...)`,
3. _examples at https://formats.kaitai.io/_ are not really useful, given that you already list a bunch of example values under the `examples` key - and in fact, an example of how to use _any feature_ of the KSY spec can be found in the format gallery, so you could write it to all keys here ;),
4. `top-level` is not an adjective here, so there is no reason to put a hyphen there `-` (it should be _at the top level_ I think)

```suggestion
          "description": "list of some identifiers that can give additional information about the format\n\nshould be written in `lowercase-kebab-case` and listed in alphabetical order\n\nexamples in existing schemas at https://formats.kaitai.io/\n\nshould be used only at the top level",
```

---

Done